src/Main.java

```diff
@@ -2402,11 +2466,19 @@ else if (region == 0x9) {
                 return inventory.contains(Items.STONE_FOOT) && canSwim(inventory);
             }
             else if (region == 0xa) {
+				/*
```

Comment:
This path is awesome, great find

Reply:
Was a complicated one to test! Needed to get Boots without tempting myself into getting Wheels to do so! :P